server/server.go

```diff
@@ -0,0 +1,55 @@
+package server
+
+import (
+	db "github.com/iorhachovyevhen/dsss/storage"
+	"github.com/spf13/viper"
+	"github.com/valyala/fasthttp"
+	"log"
+)
+
+var storage db.DataKeeper
```

Comment:
Storage not must be a global varibale=)  Why you don't want to use structures?=D The main idea is to separate logic and code responsibility area